id/src/test/kotlin/com/amplitude/id/FileIdentityStorageTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.amplitude.id
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class FileIdentityStorageTest {
+
+    @Test
+    fun `test FileIdentityStorage create and save success`() {
+        val identityStorageProvider = FileIdentityStorageProvider()
+        val identityConfiguration = IdentityConfiguration(
+            instanceName = "testInstance",
+            identityStorageProvider = identityStorageProvider,
+            apiKey = "test-api-key"
+        )
+        val fileIdentityStorage = identityStorageProvider.getIdentityStorage(identityConfiguration)
+        fileIdentityStorage.saveUserId("user_id")
+        fileIdentityStorage.saveDeviceId("device_id")
+        val savedIdentity = fileIdentityStorage.load()
+        val expectedIdentity = Identity("user_id", "device_id")
+        assertEquals(savedIdentity, expectedIdentity)
+    }
+
+    @Test
+    fun `test FileIdentityStorage load from file success`() {
```

Comment:
nit: I think it would be good to isolate this test to do its own set up rather than being dependent on the previous test.

Reply:
Yeah, I tried to use some other method earlier though didn't work well. Let me just extract the setup to separate method.